Makefile

```diff
@@ -36,6 +36,11 @@ test-%: test/%/out.llbc out/test-%/main.c | all
 out/%:
 	mkdir -p $@
 
+.PHONY: format
```

Comment:
can you just integrate this into `format-apply`, and try to notice missing formatting in format-check? that way it's unified for C and OCaml.

Reply:
Oh yep, I hadn't seen them